### PR TITLE
Prepare the doc for the 3.0

### PR DESCRIPTION
**Pull request by anaelChardan**
**Description**

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -60,7 +60,7 @@
 # built documents.
 #
 # The short X.Y version.
-version = os.getenv('PIM_VERSION', 'master')
+version = '3.0'
```
Comment (anaelChardan) [inline conf.py]: TO REVERT

File: cloud_edition/flexibility_mode/docs/composer_settings.rst [hunk 2]
```diff
@@ -31,12 +31,12 @@ Composer settings
         {
             "type": "vcs",
             "url": "https://github.com/akeneo/pim-community-dev.git",
-            "branch": "master"
+            "branch": "3.0"
```
Comment (damien-carcel) [inline cloud_edition/flexibility_mode/docs/composer_settings.rst]: nope, this one can stay on "master" (it is in fact probably useless).

File: cloud_edition/flexibility_mode/docs/composer_settings.rst [hunk 3]
```diff
@@ -31,12 +31,12 @@ Composer settings
...
         },
         {
             "type": "vcs",
             "url": "ssh://git@distribution.akeneo.com:443/pim-enterprise-dev-<who_i_am>.git",
-            "branch": "master"
+            "branch": "3.0"
```
Comment (damien-carcel) [inline cloud_edition/flexibility_mode/docs/composer_settings.rst]: same

File: migrate_pim/index.rst [hunk 4]
```diff
@@ -87,8 +87,8 @@ To upgrade, please change the composer.json to:
 
     {
         ...
-        "akeneo/pim-enterprise-dev": "2.3.10",
-        "akeneo/pim-community-dev": "2.3.10",
+        "akeneo/pim-enterprise-dev": "3.0.10",
```
Comment (damien-carcel) [inline migrate_pim/index.rst]: You have a 2.3 in the next paragraph.

File: migrate_pim/index.rst [hunk 5]
```diff
@@ -158,6 +159,7 @@ Here are the migration guides:
 * `From v1.1 to v1.2`_
 * `From v1.0 to v1.1`_
 
+.. _From v2.3 to v3.0: https://github.com/akeneo/pim-community-standard/blob/master/UPGRADE-3.0.md
```
Comment (jjanvier) [inline migrate_pim/index.rst]: 3.0?

File: technical_architecture/best_practices/reusable_bundle.rst [hunk 6]
```diff
@@ -56,8 +56,8 @@ You can find some configuration examples on our `Akeneo-Labs`_ projects (`scruti
 .. _Travis: https://travis-ci.org/
 .. _Scrutinizer: https://scrutinizer-ci.com/
 .. _Akeneo-Labs: https://github.com/akeneo-labs
-.. _scrutinizer.yml: https://github.com/akeneo-labs/CustomEntityBundle/blob/master/.scrutinizer.yml
-.. _travis.yml: https://github.com/akeneo-labs/CustomEntityBundle/blob/master/.travis.yml
+.. _scrutinizer.yml: https://github.com/akeneo-labs/CustomEntityBundle/blob/3.0/.scrutinizer.yml
```
Comment (jjanvier) [inline technical_architecture/best_practices/reusable_bundle.rst]: not sure they follow the same version
check with JM

File: migrate_pim/index.rst (hunk 5 above)
Comment (anaelChardan) [reply migrate_pim/index.rst]: Ah yes thanks ;)

File: technical_architecture/best_practices/reusable_bundle.rst (hunk 6 above)
Comment (anaelChardan) [reply technical_architecture/best_practices/reusable_bundle.rst]: Seen with him already ;)

File: import_and_export_data/formats/attribute.rst [hunk 7]
```diff
@@ -10,7 +10,7 @@ The import/export file follows this data structure:
     :file: examples/attribute_properties.csv
     :delim: ;
 
-.. _MeasureBundle measures: https://github.com/akeneo/MeasureBundle/blob/master/Resources/config/measure.yml
+.. _MeasureBundle measures: https://github.com/akeneo/MeasureBundle/blob/3.0/Resources/config/measure.yml
```
Comment (rybus) [inline import_and_export_data/formats/attribute.rst]: broken link

Comment (damien-carcel) [reply import_and_export_data/formats/attribute.rst]: Yep, the check of the hyperlink by Travis was removed this summer :disappointed:. We will put it back.

Comment (anaelChardan) [reply import_and_export_data/formats/attribute.rst]: Yes for now ;)

**Pull request merged**